c++/src/CMakeLists.txt

```diff
@@ -236,11 +231,11 @@ endif(BUILD_LIBHDFSPP)
 add_library (orc STATIC ${SOURCE_FILES})
 
 target_link_libraries (orc
-  protobuf
-  zlib
-  snappy
-  lz4
-  zstd
+  orc::protobuf
```

Comment:
what's the rationale of an orc prefix?

Reply:
Because of it being an alias, I have to use some prefix. Doesn't have to be orc::, could be 3rdparty:: as well, or if you have something better in mind. I am deliberately using an alias target here.